src/main/java/com/ioet/bpm/skills/domain/Category.java

```diff
@@ -0,0 +1,17 @@
+package com.ioet.bpm.skills.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import javax.validation.constraints.NotBlank;
+
+@Data
+@AllArgsConstructor
+public class Category {
+    @NotBlank
+    String name;
+    @NotBlank
+    Double businessValue;
+    @NotBlank
+    Double predictiveValue;
```

Comment:
It is not needed because Category is embedded into Skill model. So I guess what you want is that Category has its own DB table, so I have done that. Now `Category` has its own DB table and it is related to Skill through the `categoryId`.